demo/retrieve_code_from_response.ipynb

```diff
@@ -63,15 +89,42 @@
     "            temp[i] = \"\"\n",
     "        response = \"\".join(temp)\n",
     "    \n",
-    "    return response"
+    "    return response\n",
+    "\n",
+    "def extract_python2(response):\n",
```

Comment:
I would be happy if there were no duplicate functions with almost the same name. That prevents confusion.

Reply:
Fixed in new PR, removing functions extract_python1, extract_python2, and consolidated to just "extract_python"